Magento_Catalog/templates/product/list/items.phtml

```diff
@@ -0,0 +1,308 @@
+                                    >
+                                        <?= $block->escapeHtml($_item->getName()) ?>
+                                    </a>
+                                </div>
+
+                                <div class="catalog-grid-item__price">
+                                    <?= /* @escapeNotVerified */ $block->getProductPrice($_item) ?>
+                                </div>
+                            </div>
+
+                            <?php if ($showAddTo || $showCart): ?>
+                                <div class="catalog-grid-item__actions">
+                                    <?php if ($showCart): ?>
+                                        <?php if ($_item->isSaleable()): ?>
+                                            <?php if ($_item->getTypeInstance()->hasRequiredOptions($_item)): ?>
+                                                <button
+                                                    class="
+                                                        button
+                                                        catalog-grid-item__primary-action
+                                                        tocart
+                                                        primary
+                                                    "
+                                                    data-mage-init='{
+                                                        "redirectUrl": {
+                                                            "url": "<?= /* @escapeNotVerified */ $block->getAddToCartUrl($_item) ?>"
+                                                        }
+                                                    }'
+                                                    type="button"
+                                                    title="<?= /* @escapeNotVerified */ __('Add to Cart') ?>"
+                                                >
+                                                    <?= /* @escapeNotVerified */ __('Add to Cart') ?>
+                                                </button>
+                                            <?php else: ?>
+                                                <?php
+                                                $postDataHelper = $this->helper('Magento\Framework\Data\Helper\PostHelper');
+                                                $postData = $postDataHelper->getPostData($block->getAddToCartUrl($_item), ['product' => $_item->getEntityId()])
+                                                ?>
+                                                <button
+                                                    class="
+                                                        button
+                                                        catalog-grid-item__primary-action
+                                                        tocart
+                                                        primary
```

Comment:
can you check if we need `primary` class

Reply:
nope, it's not necessary, deleted